src/systems/systems/src/hydraulic/mod.rs

```diff
@@ -2697,6 +2997,24 @@ mod tests {
 
     use super::*;
 
+    struct TestFluid {
+        is_hot: bool,
+    }
+    impl TestFluid {
+        fn overheat() -> Self {
+            Self { is_hot: true }
+        }
+
+        fn nominal() -> Self {
+            Self { is_hot: true }
+        }
```

Comment:
```suggestion
        fn overheat() -> Self {
            Self { is_hot: true }
        }

        fn nominal() -> Self {
            Self { is_hot: false }
        }
```
I think this is correct? Otherwise both function do the same

Reply:
Oh yeah good catch!